Chatter/AvatarPicker/AvatarPickerCell.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  AvatarPickerCell.swift
+//  Chatter
+//
+
+import UIKit
+
+class AvatarPickerCell: UICollectionViewCell {
+
+    private var avatarImageView: UIImageView = {
+        let imageView = UIImageView()
+        return imageView
+    }()
+
+    override init(frame: CGRect = .zero) {
```

Comment:
I'm assuming that you never set the frame to something other than `.zero`, so I would remove the parameter at all.